gofedlib/providers/data/ip2pp_mapping.json

```diff
@@ -24,13 +24,11 @@
 {"ipprefix": "gopkg.in/v1/tomb","provider_prefix":"github.com/go-tomb/tomb"},
 {"ipprefix": "gopkg.in/tomb.v2","provider_prefix":"github.com/go-tomb/tomb"},
 {"ipprefix": "gopkg.in/v2/tomb","provider_prefix":"github.com/go-tomb/tomb"},
-{"ipprefix": "gopkg.in/yaml.v2","provider_prefix":"github.com/go-yaml/yaml"},
-{"ipprefix": "gopkg.in/v2/yaml","provider_prefix":"github.com/go-yaml/yaml"},
 {"ipprefix": "gopkg.in/gcfg.v1","provider_prefix":"github.com/go-gcfg/gcfg"},
 {"ipprefix": "gopkg.in/olivere/elastic.v2","provider_prefix":"github.com/olivere/elastic"},
 {"ipprefix": "k8s.io/heapster","provider_prefix":"github.com/kubernetes/heapster"},
 {"ipprefix": "k8s.io/kubernetes","provider_prefix":"github.com/kubernetes/kubernetes"},
 {"ipprefix": "gopkg.in/natefinch/lumberjack.v1","provider_prefix":"github.com/natefinch/lumberjack"},
 {"ipprefix": "launchpad.net/gocheck","provider_prefix":"github.com/go-check/check"},
-{"ipprefix": "speter.net/go/exp/math/dec/inf","provider_prefix":"github.com/hankya/go-decimal-inf.exp"}
+{"ipprefix": "speter.net/go/exp/math/dec/inf","provider_prefix":"code.google.com/p/go-decimal-inf"}
```

Comment:
code.google.com/p/go-decimal-inf is no longer supported, thus the reason for using github repository